sql/core/src/test/resources/sql-tests/results/pgSQL/aggregates_part3.sql.out

```diff
@@ -0,0 +1,108 @@
+-- Automatically generated by SQLQueryTestSuite
+-- Number of queries: 11
+
+
+-- !query 0
+create temporary view varchar_tbl as select * from values
+  ('a'),
+  ('A'),
+  ('1'),
+  ('2'),
+  ('3'),
+  (''),
+  ('c')
+  as varchar_tbl(f1)
+-- !query 0 schema
+struct<>
+-- !query 0 output
+
+
+
+-- !query 1
+select concat_ws(',', collect_list(a)) from (values('aaaa'),('bbbb'),('cccc')) g(a)
+-- !query 1 schema
+struct<concat_ws(,, collect_list(a)):string>
+-- !query 1 output
+aaaa,bbbb,cccc
+
+
+-- !query 2
+select concat_ws(',', collect_list(a)) from (values('aaaa'),(null),('bbbb'),('cccc')) g(a)
+-- !query 2 schema
+struct<concat_ws(,, collect_list(a)):string>
+-- !query 2 output
+aaaa,bbbb,cccc
+
+
+-- !query 3
+select concat_ws('AB', collect_list(a)) from (values(null),(null),('bbbb'),('cccc')) g(a)
+-- !query 3 schema
+struct<concat_ws(AB, collect_list(a)):string>
+-- !query 3 output
+bbbbABcccc
+
+
+-- !query 4
+select concat_ws(',', collect_list(a)) from (values(null),(null)) g(a)
+-- !query 4 schema
+struct<concat_ws(,, collect_list(a)):string>
+-- !query 4 output
+
+
+
+-- !query 5
+select concat_ws(',', sort_array(array_distinct(collect_list(f1)))) from varchar_tbl
+-- !query 5 schema
+struct<concat_ws(,, sort_array(array_distinct(collect_list(f1)), true)):string>
+-- !query 5 output
+,1,2,3,A,a,c
```

Comment:
this result is different from pg? https://github.com/postgres/postgres/blob/REL_12_BETA2/src/test/regress/expected/aggregates.out#L1462

Reply:
Revert it to original query.